client/web/src/repo/commits/RepositoryCommitsPage.tsx

```diff
@@ -138,6 +128,38 @@ export const RepositoryCommitsPage: React.FunctionComponent<React.PropsWithChild
     const repo = props.repo
     const filePath = props.match.params.filePath
 
+    const { connection, error, loading, hasNextPage, fetchMore } = useShowMorePagination<
+        RepositoryGitCommitsResult,
+        RepositoryGitCommitsVariables,
+        GitCommitFields
+    >({
+        query: REPOSITORY_GIT_COMMITS_QUERY,
+        variables: {
+            repo: repo?.id,
```

Comment:
```suggestion
            repo: repo.id,
```
It seems `repo` can't be `undefined` anymore.